src/Deserialization/FromBytes.js

```diff
@@ -1,27 +1,29 @@
-/* global require exports BROWSER_RUNTIME */
+/* global BROWSER_RUNTIME */
 
-var lib;
-if (typeof BROWSER_RUNTIME != 'undefined' && BROWSER_RUNTIME) {
-    lib = require('@emurgo/cardano-serialization-lib-browser');
+let lib;
+if (typeof BROWSER_RUNTIME != "undefined" && BROWSER_RUNTIME) {
+  lib = require("@emurgo/cardano-serialization-lib-browser");
 } else {
-    lib = require('@emurgo/cardano-serialization-lib-nodejs');
+  lib = require("@emurgo/cardano-serialization-lib-nodejs");
 }
 
-const fromBytes = name => helper => bytes => {
-    try {
-        return helper.valid(lib[name].from_bytes(bytes));
-    } catch (e) {
-        return helper.error(name + ".from_bytes() raised " + e);
-    }
+const fromBytes = (name) => (helper) => (bytes) => {
```

Comment:
I don't like this style, can we remove the parentheses?

Reply:
OK, that can be configured